src/common/roll_renderer.js

```diff
@@ -326,7 +326,7 @@ class Beyond20RollRenderer {
         if (request.sendMessage && this._displayer.sendMessage)
             this._displayer.sendMessage(request, title, html, buttons, character, request.whisper, play_sound, source, attributes, description, attack_rolls, roll_info, damage_rolls, total_damages, open)
         else
-            this._displayer.postHTML(request, title, html, buttons, character, request.whisper, play_sound);
+            this._displayer.postHTML(request, title, html, buttons, character, request.whisper, play_sound, attack_rolls, damage_rolls);
```

Comment:
I would prefer if all the other arguments get sent as well, since they are available and they might be needed at some point in the future, so I'd prefer if the arguments to `sendMessage` and `postHTML` would just be identical at this point.

Reply:
Fixed